lib/ui/window/platform_configuration.cc

```diff
@@ -377,7 +377,18 @@ void PlatformConfiguration::BeginFrame(fml::TimePoint frameTime,
   }
   tonic::DartState::Scope scope(dart_state);
 
-  int64_t microseconds = (frameTime - fml::TimePoint()).ToMicroseconds();
+  int64_t microseconds = frameTime.ToEpochDelta().ToMicroseconds();
```

Comment:
nit: mark as `const`

Reply:
actually; it has to be non-const so I can clamp it to the greater of microseconds or last_microseconds.